.jenkins/infrastructure/dockerfiles/linux/Dockerfile.minimal

```diff
@@ -0,0 +1,26 @@
+# Copyright (c) Open Enclave SDK contributors.
+# Licensed under the MIT License.
+
+ARG ubuntu_version=18.04
+
+FROM ubuntu:${ubuntu_version}
+
+ARG UNAME=jenkins
+ARG GNAME=jenkins
+ARG UID=1000
+ARG GID=1000
+
+# Install essential packages
+RUN apt-get update
+RUN apt-get install --no-install-recommends -y lsb-release wget ca-certificates git
+
+# Run Ansible
+COPY ./ /oe
+RUN /oe/scripts/ansible/install-ansible.sh
```

Comment:
Is there any significant benefit to using ansible within docker instead of just using apt-get?

Reply:
It's best practice to leave container builds as simple as possible (e.g. without Ansible) but factoring out Ansible wasn't a consideration in the scope of this change. I do want to further optimize these docker builds, but it will have to be done separate to this change.